sdk/core/core-client/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.5.1 (Unreleased)
+## 1.5.1 (2022-03-31)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added a new property endpoint in ServiceClientOptions and mark the baseUri as deprecated to encourage people to use endpoint. 
```

Comment:
Can you add a link to the PR in this changelog entry? Also, can you set the date to Unreleased?